src/handlebars/views/report.hbs

```diff
@@ -13,7 +24,13 @@
       <input type="hidden" id="avatarUrl" name="avatarUrl" />
 
       <!-- location input -->
-      <button class="button form-button" id="getLocationButton" type="button" name="location">Submit Location</button>
+      <div class="button form-button expanded-location" id="provideLocation">
+
+        <button class="button form-button" id="searchLocation" type="button" name="">Search Location</button>
+
+        <button class="button form-button" id="getLocationButton" type="button" name="location">Use Current Location</button>
+
```

Comment:
might be good to create a hidden input to hold the coords - same as we've got for the image url on line 24 - then when we process the form we only need to get the value from the one element